### PR TITLE
Faraday

### PR DESCRIPTION
**Pull request by sgkang**
Pull request for "Effects of topo"

### CODE REVIEW

File: Untitled.ipynb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython2",
+   "version": "2.7.10"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 0
+}
```
Comment (lheagy) [inline Untitled.ipynb]: Can you give this notebook a descriptive name???

File: content/maxwell1_fundamentals/index.rst [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .. _maxwell1_fundamentals_index:
 
-ME I: Fundamentals
+EM I: Fundamentals
```
Comment (lheagy) [inline content/maxwell1_fundamentals/index.rst]: Lets stick with ME I for now (Maxwell's equations 1)

File: content/introduction/index.rst [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 Introduction
 ============
+sss
+.. youtube:: http://www.youtube.com/watch?v=Ql9sn3aLLlI
```
Comment (lheagy) [inline content/introduction/index.rst]: as much as I love climbing videos, not sure what this has to do with EM :)

File: content/maxwell2_steady_state/SphericalDepression_code.rst [hunk 4]
```diff
@@ -0,0 +1,8 @@
+.. _effecttopo_code:
```
Comment (lheagy) [inline content/maxwell2_steady_state/SphericalDepression_code.rst]: do you guys want this included in a table of contents somewhere? or only a link to it?

**Comment by lheagy** [comment]
The merge conflicts need to be fixed before we can merge this pull request.

File: Untitled.ipynb (hunk 1 above)
Comment (sgkang) [inline Untitled.ipynb]: We got rid of that. That was dummy notebook

File: content/introduction/index.rst (hunk 3 above)
Comment (sgkang) [inline content/introduction/index.rst]: Sorry I was playing with personal things

File: content/maxwell2_steady_state/SphericalDepression_code.rst (hunk 4 above)
Comment (sgkang) [inline content/maxwell2_steady_state/SphericalDepression_code.rst]: We prefer a link at the moment, but we could think about table next time.

Comment (lheagy) [inline content/maxwell2_steady_state/SphericalDepression_code.rst]: kk, sounds good!

File: content/maxwell1_fundamentals/fundamental_laws/index.rst [hunk 5]
```diff
@@ -10,6 +10,5 @@ Contents:
     faraday
     ampere_maxwell
     conservation_of_charge
-    boundary_conditions
```
Comment (lheagy) [inline content/maxwell1_fundamentals/fundamental_laws/index.rst]: lets leave the bc page in this index

**Pull request merged**